doc/intro.md

```diff
@@ -113,6 +112,7 @@ Some parameters in the model can be specified by the user. The parameters, with
 * Start & end years of year range to average over when projecting domestic out-migrants; migration out of an area (2003-2014)
 * Start & end years of year range to average over when projecting international in-migrants; migration into an area (2003-2014)
 * Start & end years of year range to average over when projecting international out-migrants; migration out of area (2003-2014)
+* Brexit parameter (1) **N.B.**: Takes a value 0-1. A value of one maintains the current estimated international in-migrant population projection.  
```

Comment:
It was supposed to  just a description of what is happening and why a value of 0 to 1 should be given. Other values beyond 1 could be given, but that wouldn't make sense in terms of Brexit. It isn't coded to be restricted.